tests/testutil/service.go

```diff
@@ -202,6 +203,76 @@ func (ts *TestService) GetRunningPods() string {
 	return out
 }
 
+func (ts *TestService) GetEndpointUrl() string {
+	err := ts.reloadApp()
+	if err != nil {
+		ts.T.Fatalf("Failed to reload app:  %v", err.Error())
+		return ""
+	}
+	url, err := ts.waitForEndpointDNS()
+	if err != nil {
+		ts.T.Fatalf("Failed to get the endpoint url:  %v", err.Error())
+		return ""
+	}
+	return url
+}
+
+func (ts *TestService) GetKubeEndpointUrlByRevision(revision string) string {
+	err := ts.reloadApp()
+	if err != nil {
+		ts.T.Fatalf("Failed to reload app:  %v", err.Error())
+		return ""
+	}
+	args := append([]string{"--format", "json",
```

Comment:
Don't need append here, just do `args := []string....`. There was some leftover code that did this by accident that is being removed.